password.html

```diff
@@ -0,0 +1,2 @@
+<!-- NOTE: doneCallback here is in the scope of bedrock-angular-authn/login-modal-component -->
+<br-authn-password br-on-login="$ctrl.doneCallback(identity)"></br-authn-password>
```

Comment:
This is pretty weird, but we can leave it for now in the interest of expediency. `doneCallback` is also fairly nondescript so we'll want to revisit that.